src/pages/index.tsx

```diff
@@ -95,7 +95,8 @@ export default function Home(): JSX.Element {
           <Msg id={messageIds.home.welcome} />
         </Typography>
         {/* TODO: remove stuff below */}
-        {process.env.NODE_ENV === 'development' && (
+        {(process.env.NODE_ENV === 'development' ||
+          process.env.NEXT_PUBLIC_APP_ENV === 'development') && (
```

Comment:
This is necessary because the frontend is built in a static way. Thus the environment is passed during `RUN NEXT_PUBLIC_APP_ENV=development yarn build`.

The static build was suggested by some of my fellow developers to speed up starting the frontend :innocent: 

But I can just remove it.. scope creeping is bad.. I just wanted a quick, dockerized shortcut.